### ISSUE TITLE
multi-page quickmarks open newtab pages

### ISSUE THREAD

**Issue by behrmann** [cmcaine/tridactyl#371]
On the current beta (commit 742) opening a quickmark with multiple pages opens newtab pages as well.

Binding, e.g. `quickmark d <url 1> [ | <url 2> | ... ] | <url n>` and then using that quickmark with `gnd` opens tabs for all `n` urls, but also `n-1` newtab pages in between, i.e. tab for url 1, newtab page, tab for url 2, newtab page, ...

This is a bit annoying, since on the newtab pages the url bar grabs focus.

**Comment by bovine3dom** [comment]
I'm about 95% sure this is because you are adding pipes where you do not need any.

It should be `quickmark d url1 url2 url3`.

**Comment by behrmann** [comment]
Yeah, it was just the general being used to pipes as seperators for multiple urls from pentadactyl again.

It might be a good idea to use a canonical representation internally and accept either whitespace-separated or pipe-separated lists from users.
